src/render/activity.html

```diff
@@ -2,71 +2,170 @@
+				<li>Building the capacity to monitor the environmental state in the city of Cartagena together with the Environmental Guardians.</li>
+			</ol>
 		</p>
+		<p>
+			Video documentation of interviews with youth in Cartagena can be found on the site <a href="http://cuentalo.org/">Cuentalo.org</a> as well as on <a href="https://www.youtube.com/channel/UC75JefonQM2xbQdGM78T_Aw">YouTube</a>.
+		</p>
+
 	</div>
 
 	<div class="sjrk-activities-content">
+		<h3 class="sjrk-activities-content-header">Instituto Tecnol&oacute;gico y de Estudios Superiores de Monterrey (ITESM)</h3>
 		<p>
-			<u>Instituto Tecnol&oacute;gico y de Estudios Superiores de Monterrey (ITESM)</u><br />
-			Instituto Tecnol&oacute;gico y de Estudios Superiores de Monterrey (ITESM) provides <a href="http://sitios.itesm.mx/prepanet/">online educational programs</a> that allow youth and adults to return to school. A major benefit of these programs is that they can be completed at home on a personalised schedule that allows students to keep their jobs while going to school. ITESM works with companies to sponsor their online high school program; sponsoring companies provide internet access, computers and other incentives for youth. However, in many cases students are unable to complete online courses because they are not provided with the necessary training in digital literacy and/or they experience other barriers to learning. A lack of equipment and infrastructure and geographical isolation introduce additional challenges. ITESM is working with the SJRK project to source and develop online resources to meet the needs of students, and particularly those with learning differences, in order to support their access to online education.
+			Update Coming Soon!
 		</p>
 	</div>
 
 	<div class="sjrk-activities-content">
+		<h3 class="sjrk-activities-content-header">SMARTlab/ Inclusive Design Research Centre of Ireland at UCD</h3>
 		<p>
-			<u>SMARTlab/ Inclusive Design Research Centre of Ireland at UCD</u><br />
-			SMARTlab/ Inclusive Design Research Centre of Ireland at UCD is targeting young people with learning differences in environments outside of formal education systems in Rwanda, in order to understand the interactions among young people with learning differences, aspects of their day to day life, and their participation in society including education and youth organisations. Through informal interviews and questionnaires, data is being gathered from both young people with learning differences and the organisations that work with them.
+			In collaboration with <a href="http://uwezoyouth.org/">UWEZO Youth Empowerment</a>, other youth organizations, and young people, SMARTlab PhD student Aniyamuzaala James Rwampigi carried out a study in Rwanda from November to December 2017 to discover the mismatches between young people with learning difference and Rwanda’s formal education system, understand the reality of their lives and work, and make recommendations to match the mismatch.
```

Comment:
This sentence feels a little awkward to me. At a minimum, we should say 'learning differences' instead of 'learning difference'.